apps/api/src/database/repositories/leaderboard-repository.ts

```diff
@@ -99,7 +68,7 @@ export async function getGlobalStats(type: CompetitionType): Promise<{
     .where(inArray(votes.competitionId, relevantCompetitionIds));
 
   return {
-    activeAgents: activeAgentsResult[0]?.count ?? 0,
```

Comment:
Since active agents only make sense on ongoing competitions, this information does not make sense now. Keeping it 0 so not to break any clients